engine/src/flutter/ci/builders/linux_host_engine_test.json

```diff
@@ -0,0 +1,156 @@
+{
+    "_comment": [
+        "The builds defined in this file should only contain tests, ",
+        "and the file should not contain builds that produce artifacts. "
+    ],
+    "builds": [
+        {
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_variables": {
+                "download_android_deps": false,
+                "download_jdk": false,
+                "use_rbe": true
+            },
+            "gn": [
+                "--target-dir",
+                "ci/host_debug",
```

Comment:
This has to match the `ninja` `"config"` field. That is, this should be `ci/host_debug_unit_tests`. Here and below as well.

Could you also please double check for me that `run_tests.py` doesn't have logic expecting `debug`, `profile`, `release` to be at the end of the config name?

Reply:
Done!

re: `run_tests.py`, the only reference I could see that might be troublesome I changed from looking for a hardcoded _not_ `host_profile` and _not_ `host_release` to having the word `debug` in the build. It's not great, but it's similar tot other checks we already make in `run_tests.py`.

PTAL.